src/modules/qt/transition_qtblend.cpp

```diff
@@ -184,14 +184,17 @@ static int get_image( mlt_frame a_frame, uint8_t **image, mlt_image_format *form
 	// Check if we have transparency
 	int request_width = b_width;
 	int request_height = b_height;
+	bool imageFetched = false;
 	if ( !hasAlpha || *format == mlt_image_rgba )
 	{
 		// fetch image
 		error = mlt_frame_get_image( b_frame, &b_image, format, &b_width, &b_height, 0 );
-		if ( *format == mlt_image_rgba || mlt_frame_get_alpha( b_frame ) )
+		bool imageFetched = true;
+		if ( !hasAlpha && ( *format == mlt_image_rgba || mlt_frame_get_alpha( b_frame ) ) )
 		{
 			hasAlpha = true;
 		}
+		hasAlpha = hasAlpha && mlt_image_rgba_opaque( b_image, b_width, b_height );
```

Comment:
Shouldn't this be "! mlt_image_rgba_opaque"?

Reply:
Yes

Reply:
Thanks for noticing - I did this after waking up in the middle of the night, not the best moment for such stuff.